lib/pure/times.nim

```diff
@@ -834,8 +834,17 @@ proc initTime*(unix: int64, nanosecond: NanosecondRange): Time =
 proc nanosecond*(time: Time): NanosecondRange =
   ## Get the fractional part of a ``Time`` as the number
   ## of nanoseconds of the second.
+  ##
+  ## To get the seconds (since ``1970-01-01T00:00:00Z``) of a ``Time``,
+  ## use ``toUnix`` proc.
   time.nanosecond
 
+proc seconds*(t: Time): int64 {.benign, tags: [], raises: [], noSideEffect.} =
+  ## Convert ``t`` to a unix timestamp (seconds since ``1970-01-01T00:00:00Z``).
+  ##
+  ## This is an alias for `toUnix` proc.
+  t.seconds
```

Comment:
Won't this result in an infinite recursive call?

Reply:
No, it's an idiom in fact.

Reply:
> Won't this result in an infinite recursive call?

See the existing `nanosecond` proc above.